app/js/ZipManager.js

```diff
@@ -4,8 +4,11 @@ const DiffGenerator = require('./DiffGenerator')
 const DocumentUpdaterManager = require('./DocumentUpdaterManager')
 const DocstoreManager = require('./DocstoreManager')
 const PackManager = require('./PackManager')
-const yazl = require('yazl')
+const fs = require('fs')
 const util = require('util')
+const yazl = require('yazl')
+const tempy = require('tempy')
```

Comment:
We don't use tempy anywhere else in the codebase, and Node provides an interface to the [mkdtemp](https://nodejs.org/docs/latest-v12.x/api/fs.html#fs_fs_mkdtemp_prefix_options_callback) libc function, which is just as easy to use. [This blog post](https://advancedweb.hu/secure-tempfiles-in-nodejs-without-dependencies/) has a good discussion on that topic.

Reply:
`tempy` is really good, and it handles cleaning up the temporary files at the end of the process. I'm using it in the `web` code as well.

Reply:
I'll have a go at rewriting this to use the linked `withTempDir` function, that looks reasonably.

Reply:
Done